wren-launcher/commands/launch.go

```diff
@@ -170,9 +170,24 @@ func Launch() {
 			panic("Timeout")
 		}
 
-		// check if WrenAI is started
-		err = utils.CheckWrenAIStarted(url)
+		// check if ui is ready
+		err := utils.CheckWrenUIStarted(url)
 		if err == nil {
+			fmt.Println("WrenUI is Ready")
+			break
+		}
+		time.Sleep(5 * time.Second)
+	}
+
+	for {
+		if time.Now().After(timeoutTime) {
+			panic("Timeout")
+		}
+
+		// check if ai service is ready
+		err := utils.CheckAIServiceStarted(projectDir)
+		if err == nil {
+			fmt.Println("AI Service is Ready")
```

Comment:
Consider using `pterm.Info.Println` instead.

Reply:
delivered in 38544ea2cca6541252f3c7c738bd28b0d0526e93